official/keras_application_models/benchmark_main.py

```diff
@@ -64,30 +71,31 @@ def run_keras_model_benchmark(_):
   if FLAGS.use_synthetic_data:
     tf.logging.info("Using synthetic dataset...")
     dataset_name += "_Synthetic"
-    train_num_images = FLAGS.batch_size
-    val_num_images = FLAGS.batch_size
     train_dataset = dataset.generate_synthetic_input_dataset(
-        FLAGS.model, train_num_images)
+        FLAGS.model, FLAGS.batch_size)
     val_dataset = dataset.generate_synthetic_input_dataset(
-        FLAGS.model, val_num_images)
+        FLAGS.model, FLAGS.batch_size)
   else:
     raise ValueError("Only synthetic dataset is supported!")
 
   # If run with multiple GPUs
+  # If eager mode is enabled, only one GPU is utilized even multiple GPUs
+  # are provided
```

Comment:
Do you want to raise a warning or error message if eager execution is enabled and we use multi_gpu_model?

Reply:
Sure, I will add a warning message here. Then it keeps consistent with the help message in the --eager flag.